### DISCUSSION TITLE
input parameter with Form not work well

### DISCUSSION THREAD

**Discussion by xiaoyaolangzhi** [fastapi/fastapi#11919] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
@app.post("/test")
def test(value: str = Form("5e-05")):
    print("value: " + value)
```

### Description

The 'value' parameter should receive a string value, not a float.
![image](https://github.com/user-attachments/assets/dc740ad5-ccec-484c-ab42-0022d9f8968c)
![image](https://github.com/user-attachments/assets/930df85a-8805-4ea2-9c7c-39da3b381c85)

### Operating System

Linux

### FastAPI Version

0.111.0

### Pydantic Version

2.5.3

### Python Version

3.10.12

**Comment by YuriiMotov** [comment, marked as answer]
If you try sending such request using Swagger, you will see the curl command it uses:
```
curl -X 'POST' \
  'http://localhost:8000/test' \
  -H 'accept: application/json' \
  -H 'Content-Type: application/x-www-form-urlencoded' \
  -d 'value=0.00005'
```

And, in this command the value is already `0.00005`. So, this is likely the issue of Swagger UI.
Would you like to look for such issue on Swagger UI repo and open if not exist?
